src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,6 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
+  const item = Array.from(String(number), Number);
+  return item.reverse();
```

Comment:
You do not have to assign the Array.from method to a variable, as you are not using it anywhere else in the function, you could just add the reverse to the end of it, and delete the return and the {}, and it can be written in one line.